### DISCUSSION TITLE
Stop CW from send when hit ESC key

### DISCUSSION THREAD

**Discussion by vazquezrodjoel** [foldynl/QLog#513] [Q&A]
Is it possible to somehow setup ESC as shortcut key when TX (sending) CW to stop/halt the action/TX?  But only when TX CW, not regular since the ESC is for clearing the New Contact window.

**Comment by foldynl** [comment]
Currently, there is a shortcut for Halting CW (ALT+t). I don't see it as user-friendly to have a shortcut that has different functions depending on whether I'm sending CW or not. It can also happen that you press ESC right at the end of the session - what happens? the QSO is deleted or the transmission is terminated. 

The only thing that could be considered is to define a user-defined shortcut for Halt (instead of ALT+t)

**Reply by vazquezrodjoel** [reply]
yes, that would be acceptable...  Also maybe could be considered a cw macro, so we can setup one of the F1-7 to do that.  Just another option...

Ill give the alt-t a try... just hope I remember the combination :D
